src/lib/fragments/Circos.react.js

```diff
@@ -264,7 +264,7 @@ export default class Circos extends Component {
         } = this.props;
 
         return (
-            <div id={id} style={style} eventDatum={eventDatum}>
+            <div id={id} style={style} data-event-datum={eventDatum}>
```

Comment:
```suggestion
            <div id={id} style={style} eventdatum={eventDatum}>
```
I think we can just set this to `eventdatum` like the warning suggests, for clarity.